### PR TITLE
Merge Pull Request Dev

### LINKED ISSUE #1: Build initial integration
**Endpoints**

Because these first two require a specific issue # to be added, we were probably want to start at one and count up until an error, `null`, or some other indication that there are no more issues occurs.
- [Get Issue Data](https://github.com/ZenHubIO/API#get-issue-data)
- [Get Issue Events](https://github.com/ZenHubIO/API#get-issue-events)

- [Get Zenhub Board Data](https://github.com/ZenHubIO/API#get-the-zenhub-board-data-for-a-repository)
- [Get Epics for a Repo](https://github.com/ZenHubIO/API#get-issue-data)
- [Get Epic Data](https://github.com/ZenHubIO/API#get-issue-data)

### PR DESCRIPTION
**Pull request by benjamingregory**
Will close #1 if passes

### CODE REVIEW

File: lib/ZenHubStream.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    this.method = method;
+    this.repo_id = repo_id;
+    this.zh = new ZenHub(apiKey);
+    this.issueNumber = 1;
+  }
+
+  getChunk(method, zh, repo_id, issueNumber) {
+    return new Promise((resolve, reject) => {
+      switch (method) {
```
Comment (trong-le) [inline lib/ZenHubStream.js]: @noahbass Not really a fan of the switch statements, I think it'd be best to break them out into their own functions and return the result of the functions, whether it be null or the body. [example here](https://github.com/aries-data/aries-activity-google-places-source/blob/master/lib/index.js#L13-L15)

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        return this.getStream(config);
+      } else {
+        return await this.getRequest(config);
+      }
+    }
+
+    getRequest({ method, apiKey, repo_id, epic_id }) {
+      let zh = new ZenHub(apiKey);
+      return new Promise((resolve, reject) => {
+        switch (method) {
```
Comment (trong-le) [inline lib/index.js]: Same as comment in ZenhubStream.js

File: lib/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { Activity, singleS3StreamOutput } from 'aries-data';
+import rp from 'request-promise';
+import ZenHub from 'node-zenhub';
+import ZenHubStream from './ZenHubStream.js';
+
+export default class ZenhubSource extends Activity {
+    static props = {
```
Comment (trong-le) [inline lib/index.js]: We don't use these static props anymore so you can get rid of this object

File: README.md [hunk 4]
```diff
@@ -1 +1,51 @@
...
+
+# Aries Integration for ZenHub
+
+This is an integration to fetch data from the [ZenHub API](https://github.com/ZenHubIO/API).
+
+## Config
+
+### Api Key
+
+Your ZenHub API key:
```
Comment (trong-le) [inline README.md]: For each of these configs, can you specify where we're able to get these from?

File: lib/ZenHubStream.js (hunk 1 above)
Comment (noahbass) [inline lib/ZenHubStream.js]: Agreed @trong-le, but I think you've mentioned the wrong person 🙃

File: lib/index.js (hunk 2 above)
Comment (trong-le) [inline lib/index.js]: fixed lol. my bad @noahbass

**Comment by benjamingregory** [comment]
@undercase Have you seen this?

**Pull request merged**